packages/sdk/src/utils/constants.ts

```diff
@@ -0,0 +1,97 @@
+import { ethers } from 'ethers'
+import {
+    L1ChainID,
+    L2ChainID,
+    L1Contracts,
+    L2Contracts,
+    ContractsLike
+} from '../interfaces'
+
+// Docker
+// L1_RPC_URL = http://l1-geth:8545
+// L2_RPC_URL = http://sp-geth:4011
+
+
+const l1portalAddresses = {
+    sepolia: '0x457818a300BBa8E998889FAb2e8453BDebA238E0',
+    hardhat_local: '0x8760B3ebDFfC96C58d62D89038FfedF0ACce75Cc'
+}
+const l1StandardBridgeAddresses = {
+    sepolia: '0x06B8bF239246Cc0B08B77E743d816A690C7b1325',
+    hardhat_local: '0xb74Ec333C5Fc036df868687D5b5E8312E70ce3aA'
+}
+const l1RollupAddresses = {
+    sepolia: '0x849eB65348360a6a5f33Bce19249763fFc426D5C',
+    hardhat_local: '0x8FF3e2054223204F3Fd2C14F4cdB36cC99A5A81C'
+}
+
+export const l1ChainIds = [1337, 11155111]
+
+export const l2ChainIds = [13527]
+
+
+export const DEPOSIT_CONFIRMATION_BLOCKS: {
+    [ChainID in L2ChainID]: number
+} = {
+    [L2ChainID.SPECULAR]: 2 as const,
+    [L2ChainID.SPECULAR_HARDHAT_LOCAL]: 1 as const,
+}
+
+export const CHAIN_BLOCK_TIMES: {
+    [ChainID in L1ChainID]: number
+} = {
+    [L1ChainID.MAINNET]: 13 as const,
```

Comment:
I'm not sure this is right. These should both be 12s, right?

Reply:
We don't need them for now; just removed them.